[2021-10-06 16:20:41,567] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Pipeline.get_data1 2021-10-06T16:20:39.977699+00:00 [queued]>
[2021-10-06 16:20:41,576] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Pipeline.get_data1 2021-10-06T16:20:39.977699+00:00 [queued]>
[2021-10-06 16:20:41,577] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-10-06 16:20:41,577] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-10-06 16:20:41,578] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-10-06 16:20:41,588] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): get_data1> on 2021-10-06T16:20:39.977699+00:00
[2021-10-06 16:20:41,593] {standard_task_runner.py:52} INFO - Started process 163 to run task
[2021-10-06 16:20:41,596] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Pipeline', 'get_data1', '2021-10-06T16:20:39.977699+00:00', '--job-id', '2', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/P.py', '--cfg-path', '/tmp/tmpq02ote2p', '--error-file', '/tmp/tmphnsy_uoh']
[2021-10-06 16:20:41,596] {standard_task_runner.py:77} INFO - Job 2: Subtask get_data1
[2021-10-06 16:20:41,631] {logging_mixin.py:104} INFO - Running <TaskInstance: Pipeline.get_data1 2021-10-06T16:20:39.977699+00:00 [running]> on host 8a718b72cb7e
[2021-10-06 16:20:41,670] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=datath
AIRFLOW_CTX_DAG_ID=Pipeline
AIRFLOW_CTX_TASK_ID=get_data1
AIRFLOW_CTX_EXECUTION_DATE=2021-10-06T16:20:39.977699+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-06T16:20:39.977699+00:00
[2021-10-06 16:20:49,948] {python.py:151} INFO - Done. Returned value was: None
[2021-10-06 16:20:49,956] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=Pipeline, task_id=get_data1, execution_date=20211006T162039, start_date=20211006T162041, end_date=20211006T162049
[2021-10-06 16:20:49,979] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-10-06 16:20:50,025] {local_task_job.py:151} INFO - Task exited with return code 0
